tools/wptrunner/wptrunner/wpttest.py

```diff
@@ -484,14 +483,6 @@ def id(self):
         return self.url
 
 
-class ManualTest(Test):
-    test_type = "manual"
-
-    @property
-    def id(self):
-        return self.url
-
-
```

Comment:
What's this about?

Reply:
AFAICT it was unused code, since wptrunner can't run manual tests. But also it doesn't have a `result_cls` property, which was causing unit test failures.